kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -66,7 +66,7 @@ public class ResponsiveConfig extends AbstractConfig {
 
   public static final String STORAGE_BACKEND_TYPE_CONFIG = "responsive.storage.backend.type";
   private static final String STORAGE_BACKEND_TYPE_DOC = "The storage backend";
-  private static final StorageBackend STORAGE_BACKEND_TYPE_DEFAULT = StorageBackend.CASSANDRA;
+  private static final StorageBackend STORAGE_BACKEND_TYPE_DEFAULT = StorageBackend.MONGO_DB;
```

Comment:
oh I'm a little anxious about this, not sure whether or not existing customers set this explicitly or not. please check with them first!

I originally thought you were just going to change the default in the test configurations.

Reply:
ah sorry, good call. We should check with them though because I think we do want to shift to Mongo as the default (or do we?)

I'll extract this out into a separate PR that we can merge if/when we're ready. Does it make sense to change the default for the tests before we change the default itself?